mod-configuration-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1,4 +1,11 @@
 {
+  "scripts": [
+    {
+      "run": "after",
+      "snippetPath": "migration/config_data.sql",
+      "fromModuleVersion": "mod-configuration-5.2.0"
```

Comment:
I believe this should be the next to be released version (5.3.0), not the one already released (5.2.0).